src/main/resources/data.sql

```diff
@@ -0,0 +1,9 @@
+USE mmotodo;
+
+INSERT INTO category(id, name, description)
+VALUES  (1, 'default', '');
+
+
+INSERT INTO tag(id, name, description )
+VALUES  (1, 'none', '');
+
```

Comment:
As mentioned in the issues and other PRs, the .sql files are redundant with flyway and shouldn't be included.